src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -298,17 +298,42 @@ public void Initialize()
             }
         }
 
-        public async Task PutEditVersioningCommandAsync(JsonOperationContext context, string databaseName, BlittableJsonReaderObject val)
+        public async Task<long> ModifyDatabaseExpirationBundle(TransactionOperationContext context, string name, BlittableJsonReaderObject configurationJson)
```

Comment:
@ppekrol Yes, we need to move it to use that.
@talweiss1982 You shouldn't keep merging, you should always run on the latest code.
Those changes were made last week or the one before, they aren't new.